mkosi/__init__.py

```diff
@@ -1902,31 +1903,54 @@ def reenable_kernel_install(args: CommandLineArguments, root: str) -> None:
     make_executable(hook_path)
 
 
+def add_packages(
+    args: CommandLineArguments, packages: Set[str], *names: str, conditional: Optional[str] = None
```

Comment:
```suggestion
    args: CommandLineArguments, packages: Set[str], *names: str, *, conditional: Optional[str] = None
```
to force the usage of the keyword. I think this makes functions with varargs a bit less confusing.

Reply:
I'm pretty sure that keyword usage is already forced by `*names`. There is no way to specify `conditional` without a keyword. The syntax with `, *, ` was added to support the case where there is no `*args` parameter.

Reply:
Of course, right, without the keyword it would always be gobbled up by `names`.